pkg/glusterfs/api/types.go

```diff
@@ -28,7 +28,7 @@ const (
 	EntryStateUnknown EntryState = ""
 	EntryStateOnline  EntryState = "online"
 	EntryStateOffline EntryState = "offline"
-	EntryStateFailed  EntryState = "failed"
+	EntryStateFailed  EntryState = "removed"
```

Comment:
Please also change the variable throughout the code `EntryStateFailed` to `EntryStateRemoved` for completeness